src/twisted/test/proto_helpers.py

```diff
@@ -159,6 +162,7 @@ class StringTransport:
     """
 
     disconnecting = False
+    aborted = False
```

Comment:
Is disconnected suitable here? I'm not looking for whether the connection is actually gone, I'm looking explicitly for whether we called `abortConnection`.